Downloading Trivy binary ends in 404

Hi

When trying to use Trivy Azure Pipelines Task using the following:
```yaml
- task: trivy@1
  inputs:
    version: 0.34.0
    debug: true
    docker: false
    image: myImage:1.0
```
The following error log is shown:
```
Run requested using local Trivy binary...
Finding correct Trivy version to install...
Required Trivy version is 0.34.0
Downloading Trivy...
Downloading: https://github.com/aquasecurity/trivy/releases/download/0.34.0/trivy_0.34.0_Linux-64bit.tar.gz
##[error]Unexpected HTTP response: 404
```

When trying to download manually using the above URL, "Not Found" will also be shown.

The correct URL would probably be:
https://github.com/aquasecurity/trivy/releases/download/v0.34.0/trivy_0.34.0_Linux-64bit.tar.gz

Note the missing "v" in the URL path before the version number.

--- a/marketplace.md
+++ b/marketplace.md
@@ -116,6 +116,7 @@ jobs:
   - job: Scan the local project
     steps:
       - task: trivy@1
+        version: v0.59.0
         inputs:
           path: .
   - job: Scan the ubuntu image

